javalib/src/main/scala/java/lang/Thread.scala

```diff
@@ -1,8 +1,8 @@
 package java.lang
 
-import scalanative.unsigned._
 import scalanative.annotation.stub
-import scalanative.libc.errno
+import scalanative.meta.LinktimeInfo.isWindows
+import impl._
```

Comment:
Consider always using fully-qualified imports at the top, especially for generic names like that, which could come from other imports by accident:
```suggestion
import java.lang.impl._
```